mne_bids/tests/test_mne_bids.py

```diff
@@ -253,3 +253,82 @@ def test_cnt():
 
     cmd = ['bids-validator', '--bep006', output_path]
     run_subprocess(cmd, shell=shell)
+
+
+# iEEG Tests
+# ---------
+def test_mef3():
+    """Test raw_to_bids conversion for MEF3 data."""
+    output_path = _TempDir()
+
+    # MEF3 not yet implemented, assert we raise an error
+    __ = _TempDir()
+    data_path = op.join(__)
+    raw_fname = op.join(data_path, 'test.mef3')
+    with open(raw_fname, 'w'):
+        pass
+
+    with pytest.raises(ValueError):
+        raw_to_bids(subject_id=subject_id, session_id=session_id, run=run,
+                    task=task, acquisition=acq, raw_file=raw_fname,
+                    output_path=output_path, overwrite=True, kind='ieeg')
+
+
+def test_nwb():
+    """Test raw_to_bids conversion for Neurodata without borders data."""
+    output_path = _TempDir()
+
+    # .nwb not yet implemented, assert we raise an error
+    __ = _TempDir()
+    data_path = op.join(__)
+    raw_fname = op.join(data_path, 'test.nwb')
+    with open(raw_fname, 'w'):
+        pass
+
+    with pytest.raises(ValueError):
+        raw_to_bids(subject_id=subject_id, session_id=session_id, run=run,
+                    task=task, acquisition=acq, raw_file=raw_fname,
+                    output_path=output_path, overwrite=True, kind='ieeg')
+
+
+def test_vhdr_ieeg():
+    """Test raw_to_bids conversion for BrainVision data (ieeg)."""
+    output_path = _TempDir()
+    data_path = op.join(base_path, 'brainvision', 'tests', 'data')
+    raw_fname = op.join(data_path, 'test.vhdr')
+
+    raw_to_bids(subject_id=subject_id, session_id=session_id, run=run,
+                task=task, acquisition=acq, raw_file=raw_fname,
+                output_path=output_path, overwrite=True, kind='ieeg')
+
+    cmd = ['bids-validator', '--bep010', output_path]
+    run_subprocess(cmd, shell=shell)
```

Comment:
why not just add these two lines to the existing tests so we don't have to redefine the filenames?

Reply:
will do that (plus changing kind to iEEG)